huaweicloud/services/dcs/data_source_huaweicloud_dcs_instances.go

```diff
@@ -0,0 +1,364 @@
+			},
+			"security_group_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of the security group which the instance belongs to.`,
+			},
+			"security_group_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the name of security group which the instance belongs to.`,
+			},
+			"enterprise_project_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the enterprise project id of the dcs instance.`,
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the description of an instance.`,
+			},
+			"private_ip": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the IP address of the DCS instance.`,
+			},
+			"maintain_begin": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the time at which the maintenance time window starts.`,
+			},
+			"maintain_end": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the time at which the maintenance time window ends.`,
+			},
+			"charging_mode": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the charging mode of the cache instance.`,
+			},
+			"port": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Indicates the port of the cache instance.`,
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the cache instance status.`,
+			},
+			"used_memory": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Indicates the size of the used memory. Unit: MB.`,
+			},
+			"max_memory": {
+				Type:        schema.TypeInt,
+				Computed:    true,
+				Description: `Indicates the atotal memory size. Unit: MB.`,
+			},
+			"domain_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the domain name of the instance.`,
+			},
+			"user_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the user ID.`,
+			},
+			"user_name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the user name.`,
+			},
+			"access_user": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the username used for accessing a DCS Memcached instance.`,
+			},
+			"order_id": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: `Indicates the ID of the order that created the instance.`,
+			},
+			"tags": common.TagsSchema(),
+		},
+	}
+	return &sc
+}
+
+func resourceDcsInstanceRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+
+	var mErr *multierror.Error
```

Comment:
remove this line to line:270

Reply:
the code is automatically generated, the style is unified